GetDKAN/dkan
# Update the documentation link to the dkan sites list

fixes https://github.com/NuCivic/dkan_management/issues/311

QA

- build docs locally, confirm there is a **link** to 'dkan sites' under Introduction
- confirm there is not a 'dkan sites' **page** in the documentation
